.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Build and Test
         run: Scripts/build.swift xcode ${{ matrix.platform }} `which xcpretty`
       - name: Upload Results
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
```

Comment:
This isn't the latest version of `upload-artifacts`, but v4 introduced a breaking change around uploading to the same-named artifact multiple times that we'll need to make some changes in our scripts to support. Will do that as a follow-up.